reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -92,26 +93,79 @@ class StaticHtmlReporter : Reporter {
                     }
                 }
 
+                fun getTitleExt(mData: Map<String, String>): String {
```

Comment:
I believe the name `getTitleExt()` is confusing as a cannot make sense out of the `Ext` suffix. What does it mean?

Further `mData` seems to be an abbrevation of `metaData` which I also find hard to guess. The name should be changed to something which more meaningful but: Is that parameter actually needed? if I'm not mistaken it can just be removed so that the implementation works on `reportTableModel.metadata` directly.

Finally instead of `{ return if` please write `= if`.